server/public/model/user.go

```diff
@@ -1039,6 +1040,59 @@ type UserReport struct {
 	UserPostStats
 }
 
+func (u UserReport) GetHeaders() []string {
```

Comment:
The `GetHeaders` method is required for an object to be saved using the reporting module. It is a static method that defines the ordering and the column names for a report.